docs/dashboards/dashboard-optimization.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
@JV0812 why are you including this page? As far as I know, this does not apply to v2 dashboards.

Reply:
There was a link in the **Restricted Operators in Dashboards** doc, which refers to this page. So, I have added this page for New dashboards too.